telco-examples/mgmt-cluster/aarch64/eib/mgmt-cluster-arm-singlenode.yaml

```diff
@@ -27,46 +27,46 @@ kubernetes:
         createNamespace: true
         installationNamespace: kube-system
       - name: longhorn-crd
-        version: 104.2.0+up1.7.1
-        repositoryName: rancher-charts
+        version: 105.1.0+up1.7.2
+        repositoryName: rancher
         targetNamespace: longhorn-system
         createNamespace: true
         installationNamespace: kube-system
       - name: longhorn
-        version: 104.2.0+up1.7.1
-        repositoryName: rancher-charts
+        version: 105.1.0+up1.7.2
+        repositoryName: rancher
         targetNamespace: longhorn-system
         createNamespace: true
         installationNamespace: kube-system
-      - name: metal3
+      - name: metal3-chart
         version: 0.9.1
         repositoryName: suse-edge-upstream
         targetNamespace: metal3-system
         createNamespace: true
         installationNamespace: kube-system
         valuesFile: metal3.yaml
       - name: neuvector-crd
-        version: 104.0.1+up2.7.9
-        repositoryName: rancher-charts
+        version: 105.0.0+up2.8.3
+        repositoryName: rancher
         targetNamespace: neuvector
         createNamespace: true
         installationNamespace: kube-system
         valuesFile: neuvector.yaml
       - name: neuvector
-        version: 104.0.1+up2.7.9
-        repositoryName: rancher-charts
+        version: 105.0.0+up2.8.3
+        repositoryName: rancher
         targetNamespace: neuvector
         createNamespace: true
         installationNamespace: kube-system
         valuesFile: neuvector.yaml
       - name: rancher
-        version: 2.9.1
+        version: v2.10.1
         repositoryName: rancher-prime
         targetNamespace: cattle-system
         createNamespace: true
         installationNamespace: kube-system
         valuesFile: rancher.yaml
-      - name: rancher-turtles
+      - name: rancher-turtles-chart
```

Comment:
The name in the upstream repo is still `rancher-turtles`.

Do we also want to bump this to `0.5.0+up0.14.1`?

Reply:
tested the current version but changing the name. Thanks good catch